### PR TITLE
Added login page authentication

### PR DESCRIPTION
**Pull request by Pratikkumar24**

### CODE REVIEW

File: app/src/main/java/com/Executioner/meetx/MainActivity.java [hunk 1]
```diff
@@ -9,18 +10,30 @@
...
 import com.Executioner.meetx.authentication.Signup_page;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.analytics.FirebaseAnalytics;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
 
 public class MainActivity extends AppCompatActivity {
-    String TAG="pratiksha";
+    String TAG="mainPratik";
```
Comment (neha702) [inline app/src/main/java/com/Executioner/meetx/MainActivity.java:23]: What is this nonsense?

Comment (Pratikkumar24) [reply app/src/main/java/com/Executioner/meetx/MainActivity.java:23]: Sorry will change it

File: app/src/main/java/com/Executioner/meetx/authentication/Login_page.java [hunk 2]
```diff
@@ -34,16 +44,34 @@ protected void onCreate(Bundle savedInstanceState) {
...
-                    Log.i(TAG," UserName: "+ UserName +" Password:" + Password);
-                    Intent intent=new Intent(getApplicationContext(), homepage.class);
-                    startActivity(intent);
-                    finish();
+                    auth.signInWithEmailAndPassword(Email, Password).addOnCompleteListener(new OnCompleteListener<AuthResult>() {
+                        @Override
+                        public void onComplete(@NonNull Task<AuthResult> task) {
+                            if(task.isSuccessful())
+                            {
+                                try {
```
Comment (neha702) [inline app/src/main/java/com/Executioner/meetx/authentication/Login_page.java:61]: Try catch required?

Comment (Pratikkumar24) [reply app/src/main/java/com/Executioner/meetx/authentication/Login_page.java:61]: Ok will change it

Comment (neha702) [reply app/src/main/java/com/Executioner/meetx/authentication/Login_page.java:61]: Haan karo

**Review by neha702 — APPROVED** [review]
LGTM

**Pull request merged**